src/adapters/appnexusAst.js

```diff
@@ -225,6 +227,26 @@ function AppnexusAstAdapter() {
     return tag && tag.ads && tag.ads.length && tag.ads.find(ad => ad.rtb);
   }
 
+  function outstreamRender(bid) {
+    window.ANOutstreamVideo.renderAd({
```

Comment:
Exception would be thrown here if `window.ANOutstreamVideo` isn't defined but should be caught in `renderAd` so I guess it's alright for now.